src/lib/features/Modal.stories.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+  import { Meta, Story } from '@storybook/addon-svelte-csf';
+  import Modal from './Modal.svelte';
+  import Button from '../atoms/Button.svelte';
+  import Select from '../atoms/Select.svelte';
+  import { Close } from '../shadcnComponents/ui/dialog';
+
+  export let label = 'Select Option';
+  export let description = 'Please choose an option';
+  export let placeholder = 'Select an option...';
+  export let isRequired = false;
+  export let options = [
+    { optionLabel: 'Option 1', value: '1' },
+    { optionLabel: 'Option 2', value: '2' },
+    { optionLabel: 'Option 3', value: '3' },
+  ];
+</script>
+
+<Meta title="UI Components/Features/Modal" component={Modal} />
+
+<!--Primary-->
+<Story name="Primary Modal">
+  <Modal>
+    <Button slot="trigger">Open</Button>
+    <div slot="title">Title</div>
```

Comment:
I'm curious as to why you're using slot for everything. I could see it for body content since callers might like to add more complex stylings, but why not use props and take only text for title and description?

Reply:
It's totally a preference. The biggest reason would be in case we wanted to do different colors, then we could edit the slot directly. I'm open to moving title and description to being props.